src/controllers/actionController.js

```diff
@@ -0,0 +1,41 @@
+"use strict"
+import Validate  from '../services/validate.js';
+import Jira from'../services/jira.js';
+import GithubService from'../services/github.js';
+import jiraDTO from '../dto/jiraDTO.js';
+
+export default class ActionController{
+
+    constructor(github){
+        this.github = github
+    }
+    async createGMUD(){
+        
+        let validate = new Validate(this.github)
```

Comment:
just a tip, if you declare a value that won't change use `const` to indicate to the one who is reading that its a constant value